packages/runtime-core/__tests__/vnode.spec.ts

```diff
@@ -17,7 +19,19 @@ describe('vnode', () => {
 
   test.todo('node type/shapeFlag inference')
 
-  test.todo('cloneVNode')
+  test('cloneNode', () => {
+    const node1 = createVNode('div', { foo: 1 }, null)
+    expect(cloneVNode(node1)).toEqual(node1)
+
+    const node2 = createVNode('div', null, [node1])
+    expect(cloneVNode(node2)).toEqual(node2)
+
+    let node = null
+    for (let i = 1; i <= 100; ++i) {
```

Comment:
Not sure if this loop makes sense. If anything we should that two clones from the same original are equal.